articles/dashboard/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Management Dashboard
+description: Working with the Management Dashboard
+classes: topic-page
+---
+
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Management Dashboard</h1>
+  <p>
+    When working in the Management Dashboard, you may find the following articles helpful.
+  </p>
+</div>
+
+<ul class="topic-links">
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/dashboard/creating-users-in-the-management-portal">Create Users in the Management Dashboard</a>
```

Comment:
Add a short description after each title. Something like the doc's `description` metadata should be ok.

Reply:
Missing description. Like this it does not look good, we might as well have a list of bullets:

![image](https://user-images.githubusercontent.com/11715799/32491613-f7c4180a-c3bf-11e7-97b9-0650ad661cbf.png)